S21/ajayrazz99/wordChanger.py

```diff
@@ -0,0 +1,48 @@
+def LowerCaseConverter(word,lang):
+    lang=lang
+    word=word
+    if(lang=='turkey'or lang=='az' or lang=='tr'):      #converts uppercase to lowercase
+        for letter in word:
+            if(letter=='i'or letter=="I"):
+                I=u"\u0131"
+                print(I,end='')
+            else:
+                print(letter.lower(),end='')
+    
+    if(lang=='Greek' or lang=='el'):
+        leng=len(word)-1
+        for i in range(leng):
+            if(word[i]=="Σ"):
+                I=u"\u03C3"
+                print(I,end='')
+            else:
+                print(word[i].lower(),end='')
+        if(word[-1]=="Σ"):  
+            I=u"\u03C2"
+            print(I,end='')
+    
+    if(lang=='irish' or lang=='ga' or lang=='ga-IE' or lang=='Irish'):
+        leng=len(word)
+    
```

Comment:
May be we can consider all the vowels into an array and as we already have the word, for looping vowels array and adding '-' at desired location would be great. what do you think ?

Reply:
That is exactly what I did. But the difference is that each and every vowel is assigned to a variable.